### PR TITLE
disable attachments on private threads

### PR DESCRIPTION
**Pull request by dominictarr**
people (reasonably) assume they are encrypted, but they arn't encrypted currently.

https://github.com/ssbc/patchbay/pull/114

### CODE REVIEW

**Comment by dominictarr** [comment]
so I think we should disable them until we have them be encrypted properly.

File: modules/message/html/compose.js [hunk 1]
```diff
@@ -21,7 +21,8 @@ exports.needs = nest({
 exports.gives = nest('message.html.compose')
 
 exports.create = function (api) {
-  return nest('message.html.compose', function ({shrink = true, meta, prepublish, placeholder = 'Write a message'}, cb) {
+  return nest('message.html.compose', opts = {}, function (, cb) {
```
Comment (ahdinosaur) [inline modules/message/html/compose.js]: i think this line should be

```js
return nest('message.html.compose', function (opts, cb) {
```

Comment (dominictarr) [inline modules/message/html/compose.js]: yeah, sorry I pushed a new commit that fixes that.

**Comment by dominictarr** [comment]
that would be awesome!

**Comment by dominictarr** [comment]
definitely worth a :tada:

**Pull request closed without merging**